### PR TITLE
CON-2010-transfer-sending-employers

### PR DESCRIPTION
**Pull request by shomavg**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml [hunk 1]
```diff
@@ -39,12 +39,17 @@
...
 
-        </div>
     </div>
+
 </div>
 
 @section Back
-{     
+{
     <a id="back-link" class="govuk-back-link" href="@Url.Action("Cohorts", "Cohort", new { Model.AccountHashedId})">Back</a>
```
Comment (Howesy77) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml:54]: <div class="das-js-back-link"></div>

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml [hunk 2]
```diff
@@ -6,17 +6,17 @@
...
 
 <div class="govuk-grid-row">
-
-    <div class="govuk-grid-row">
-        <div class="govuk-width-container">
-            <h1 class="govuk-heading-xl">@Model.Title</h1>
-
+    <div class="govuk-width-container">
+        @(await Html.PartialAsync("_ApprenticeshipRequestsHeaderPartial", Model.ApprenticeshipRequestsHeaderViewModel))
+        <h1 class="govuk-heading-xl">@Model.Title</h1>
```
Comment (Howesy77) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml:11]: Don't think we need the H1

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml (hunk 1 above)
Comment (shomavg) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml:54]: This change has been done on another branch. Its in a PR.

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml (hunk 2 above)
Comment (shomavg) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml]: removed.

**Pull request merged**